lib/api/src/modules/shortcuts.ts

```diff
@@ -23,17 +65,26 @@ export const defaultShortcuts = Object.freeze({
   nextStory: ['alt', 'ArrowRight'],
   shortcutsPage: [controlOrMetaKey(), 'shift', ','],
   aboutPage: [','],
-  // This one is not customizable
-  escape: ['escape'],
+  escape: ['escape'], // This one is not customizable
 });
 
-export default function initShortcuts({ store }) {
-  const api = {
+export interface Event extends KeyboardEvent {
+  target: {
+    tagName: string;
+    addEventListener(): void;
+    removeEventListener(): boolean;
+    dispatchEvent(event: Event): boolean;
+    getAttribute(attr: string): string | null;
```

Comment:
Is it `null`? Or `undefined`? `null` is treated as a value, `undefined` is just not defined

Reply:
it DOM-api returns either `null` or `string`